thread_pool.h

```diff
@@ -32,13 +33,13 @@ class ThreadPool {
 
 private:
     RejectedJobPolicy* rejected_job_policy;
-    unsigned num_active_threads = 0;
+    unsigned num_active_threads;
```

Comment:
Why is this removed?

Reply:
> Revert the search/lobbby id change to string. It should be a number

Why ? If we keep it as an int that will only add redundant code every time we try to pass this variable in a response. This way we only parse it once when the search is created and we don't have to worry about anything later on.

Reply:
Either way we set this variable every time we initialize the Thread Pool so i do no think this has any impact whatsoever.